scripts/meson.build

```diff
@@ -0,0 +1,29 @@
+# configures `scripts/nix-profile.sh.in` (and copies the original to the build directory).
+# this is only needed for tests, but running it unconditionally does not hurt enough to care.
```

Comment:
```suggestion
# configures `scripts/nix-profile.sh.in` (and copies the original to the build directory).
# this is only needed for tests, but it's quick enough to run unconditionally.
```

Is this even relevant with our componentized build? Does it run unconditionally in the dev shell? Surely meson knows when to re-run `configure_file` or not? Or if it doesn't then surely this is normal practice and doesn't need a comment?

```suggestion
```